ui/utils.py

```diff
@@ -68,10 +76,10 @@ def query(query, filters={}, top_k_reader=5, top_k_retriever=5) -> Tuple[List[Di
                 {
                     "context": "..." + answer["context"] + "...",
                     "answer": answer.get("answer", None),
-                    "source": answer["meta"]["name"],
+                    "source": answer["type"],
                     "relevance": round(answer["score"] * 100, 2),
                     "document": [doc for doc in response["documents"] if doc["id"] == answer["document_id"]][0],
-                    "offset_start_in_doc": answer["offsets_in_document"][0]["start"],
+                    # "offset_start_in_doc": answer["offsets_in_document"][0]["start"],
```

Comment:
maybe remove instead of comment out, can always trace back in git history

Reply:
True, though I will remove it maybe in next PR, when I finish the feedback loop, to be 100% it won't be useful (if I remove it now, I will probably forget it even existed :D )